make/go/scripts/checknolintlint.bash

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+# Managed by makego. DO NOT EDIT.
+
+## checknolintlint exits with an exit code of 0 if nolintlint is enabled and configured properly.
+## It exits with an exit code of 2 if nolintlint is not enabled in .golangci.yml.
+## It exits with an exit code of 1 if nolintlint is not configured according to standards.
+
+set -euo pipefail
+
+# Check if nonolint linter is enabled in config
```

Comment:
I would probably first check if `golangci.yml` exists, and if not exit 0

Reply:
If it doesn't exist, then that means it isn't enabled (since it isn't enabled by default), so I think we should exit 1 but agree a short circuit would be good here.

Reply:
Fixed.